workitem/workitem_repository.go

```diff
@@ -587,7 +602,8 @@ func (r *GormWorkItemRepository) Save(ctx context.Context, spaceID uuid.UUID, up
 		}
 		fieldValue := updatedWorkItem.Fields[fieldName]
 		var err error
-		if fieldName == SystemAssignees || fieldName == SystemLabels || fieldName == SystemBoardcolumns {
+		// Remove fields with type list if there are no elements (eg: Assignee, Labels)
+		if fieldDef.Type.GetKind() == "list" {
```

Comment:
I cannot use `fieldDef.Type.GetKind() == workitem.KindList` because it would cause a cyclic dependency.

Reply:
But you can use `fieldDef.Type.GetKind() == KindList`. I mean you're already in the `workitem` package. So no need to import it again ;)

Reply:
See https://github.com/fabric8-services/fabric8-wit/pull/2196/commits/ee41af2ac7d8b0f356cdd96b4d854181d9883fc8

I should've seen that earlier :)